### ISSUE TITLE
Questions about the splits for argoverse2

### ISSUE THREAD

**Issue by haruishi43** [yuantianyuan01/StreamMapNet#6]
Thanks for opensourcing your project.

In your argoverse2 preprocessing script `argoverse_converter.py`, I noticed that you removed 6 logs.
It seems that there are only 4 logs that shouldn't be used for training models according to [argoverse2 user guide](https://argoverse.github.io/user-guide/api/hd_maps.html#training-online-map-inference-models).
Are there any reasons for excluding additional logs?

Also, the text files seem to include 749/149 for train/val split which is different from the 700/150 split mentioned in the paper.
Are the results obtained in the paper using the 749/149 split?
Is the test set also being used for the additional logs for new train/val split?

**Comment by yuantianyuan01** [comment]
Hi,

For the first question, the other two logs are removed due to NaN problem in map data. For the second question, we made a mistake in our paper. The new split on argoverse2 stands for the 749/149 split. Thank you for pointing it out and we will fix it in next revision.

**Comment by haruishi43** [comment]
@yuantianyuan01 thanks for your response!

Following question, why did you removed the ~100 logs for argoverse2? In the appendix it mentions that you used all of the splits (~1000 logs) to generate train/val split. Were other samples removed due to NaN problems as well?

**Comment by yuantianyuan01** [comment]
To make sure there's completely no overlap between training and validation set, we remove the logs on the borderline.
